particles/WasmTemplate/.gitignore

```diff
@@ -0,0 +1,2 @@
+example-arcs.h
```

Comment:
If these are generated/built files, maybe we should structure things so they end up in a `build` directory, to follow where we have such things elsewhere? I see later `build` is intended for tests, I suppose I just wonder whether we want to be ignoring generated source files that we may drop into non-build directories, it seems a slippery slope.

Reply:
This is still a work in progress, and the whole wasm.json "build" system is definitely not intended to be the final state. For tests, both the headers and the module file are output under build. For this example code I wanted the generated header to be placed in the source tree for visibility, so users can see the API for generated entity classes.

We could still output the module itself to build, but at the moment there's only one setting for the output dir.